src/mcsimulation.cpp

```diff
@@ -111,7 +111,7 @@ double MCSimulation::getExpectedFreeeDecay(unsigned i)
 {
     if(dataSynth){
         double b = dataSynth->getbValue(i);
-        return exp(-b*params.diffusivity);
+        return exp(-b*params.diffusivity_in);
```

Comment:
why diffusivity_in? is the default for CSF? it should be in any case the diffusivity_ex